### PR TITLE
[BN-1572]: Update package names to xyz.stratalab

### PR DESCRIPTION
**Pull request by admiraladmirable**
## Purpose
Update package names for `strata-sdk-scala`. (previously brambl-sc).

## Approach
* Update build.sbt org to `xyz.stratalab`.
* Update repository home.
* Update references of `brambl` -> `strata` (i.e. brambl-sdk -> strata-sdk).
* Update package names from `co.topl` -> `xyz.stratalab`
* Move folders to `xyz/stratalab` structure.

## Testing
CI/CD passed.

## Tickets
*BN-1572

### CODE REVIEW

File: strata-sdk/src/test/scala/xyz/stratalab/brambl/brambl/syntax/Int128SyntaxSpec.scala
Comment (mundacho) [inline strata-sdk/src/test/scala/xyz/stratalab/brambl/brambl/syntax/Int128SyntaxSpec.scala:1]: This file is still in a brambl directory, along with all others in this directory

**Review by mundacho — APPROVED** [review]
LGTM, just a minor comment on the directoy renaming.

**Review by DiademShoukralla — APPROVED** [review]
Looks great!

File: strata-sdk/src/main/scala/xyz/stratalab/strata/strata/monitoring/BifrostMonitor.scala [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 import cats.effect.IO
 import cats.effect.kernel.Resource
 import cats.implicits.{catsSyntaxParallelSequence1, toTraverseOps}
-import co.topl.brambl.dataApi.BifrostQueryAlgebra
-import co.topl.brambl.dataApi.RpcChannelResource.channelResource
-import co.topl.brambl.display.blockIdDisplay.display
+import xyz.stratalab.sdk.dataApi.BifrostQueryAlgebra
+import xyz.stratalab.sdk.dataApi.RpcChannelResource.channelResource
+import xyz.stratalab.sdk.display.blockIdDisplay.display
 import co.topl.brambl.models.transaction.IoTransaction
```
[the remark this answers is not in the document]
Comment (admiraladmirable) [reply strata-sdk/src/main/scala/xyz/stratalab/strata/strata/monitoring/BifrostMonitor.scala:9]: Yes, I'm going from leaf level projects -> protobuf specs then back to leaf level (which may need to happen all at the same time for the second pass :grimacing: )

File: strata-sdk/src/test/scala/xyz/stratalab/brambl/brambl/syntax/Int128SyntaxSpec.scala
Comment (admiraladmirable) [reply strata-sdk/src/test/scala/xyz/stratalab/brambl/brambl/syntax/Int128SyntaxSpec.scala:1]: Fixed

**Pull request merged**